<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogInWindow</class>
 <widget class="QMainWindow" name="LogInWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sarah Your Personal Dietician</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">*{
	font-family:century gothic;
	font-size: 24px;
}

QFrame{
	background:rgba(0, 0, 0, 200);
	border-radius: 15px;
}

#frame_2{
	background:transparent;
}
QPushButton{
	font-size:14px;
	background:green;
	border-radius: 15px;
}

QToolButton{
	background:transparent;
	color:darkgray;
	font-size:13px;
	border-radius: 60px;
}

QToolButton:hover{
	background:transparent;
	color:white;
	font-size:13px;
	border-radius: 60px;
}

QLabel{
	color: white;
	background:transparent;
}

QPushButton{
	color:white;
	/* border-radius: 15px; */
}

QPushButton:hover{
	/*color:darkgray;*/
	background-color:#16a016;
	border: black;
	/* border-radius: 15px; */
}

QLineEdit{
	background: transparent;
	border: none;
	color: #717072;
	border-bottom: 1px solid #717072;
	font-size: 18px;

}

QCheckBox{
	font-size:13px;
	background: none;
	color:darkgray;
}

#centralwidget{
	background: url(/home/groumeliotis/project/Sarah/images/background.jpg);
}

#pushButton_forgot_pwd{
	background: transparent;
	font-size: 12px;

}
#pushButton_forgot_pwd:hover{
	background: transparent;
	font-size: 13px;
}

#label_usr{
	font-size:15px;
	color:red;
}
#label_pwd{
	font-size:15px;
	color:red;
}
#label_logo_S{
	font-size:80px;
    font-family: 'Times New Roman', Times, serif;
}

#label_logo_arah{
	font-size:16px;
    font-family: 'Times New Roman', Times, serif;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="5" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="rememberPassword">
             <property name="maximumSize">
              <size>
               <width>380</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Remember Me ?</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_singIn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Log In</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="lineEdit_username">
           <property name="maximumSize">
            <size>
             <width>450</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_usr">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_pwd">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2" alignment="Qt::AlignHCenter">
          <widget class="QFrame" name="frame_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>70</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_logo_arah">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>80</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="4" alignment="Qt::AlignRight|Qt::AlignTop">
          <widget class="QPushButton" name="pushButton_singUp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>+ Create Profile</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2" alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="pushButton_forgot_pwd">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Forgot Your Password ?</string>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="lineEdit_password">
           <property name="maximumSize">
            <size>
             <width>450</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>rememberPassword</tabstop>
  <tabstop>pushButton_singIn</tabstop>
  <tabstop>pushButton_forgot_pwd</tabstop>
  <tabstop>pushButton_singUp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
